src/open_inwoner/components/templates/components/Card/LocationCard.html

```diff
@@ -11,7 +11,7 @@ <h3 class="utrecht-heading-3 card__heading-3">{% link href=location.get_absolute
                 {% link href='tel:'|addstr:phonenumber secondary=True text=phonenumber %}
             {% endif %}
             {% if email %}
-                {% link href='mailto:'|addstr:email secondary=True text=email %}
+                {% link href='mailto:'|addstr:email secondary=True text=email extra_classes="link--mailaddress" %}
```

Comment:
`link--mailto` might be a better classname since this is in line with what type of link href this is.

```suggestion
                {% link href='mailto:'|addstr:email secondary=True text=email extra_classes="link--mailto" %}
```

Reply:
OK, will do!